com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/CobolSemanticParser.java

```diff
@@ -30,10 +30,14 @@
    *
```

Comment:
the header on those files, you may skip it for now and I will change it tomorrow for the entire project